### PR TITLE
Change the score and health bars

### PR DESCRIPTION
**Pull request by yuhaozhang666**

### CODE REVIEW

File: src/edu/ucsb/cs56/projects/games/fish_animation/Fish.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import java.awt.geom.Line2D; 
 import java.awt.geom.PathIterator;
 import java.awt.geom.AffineTransform;
+import java.awt.geom.Ellipse2D;
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/Fish.java]: Is this used?  If it's not you should remove it.

File: src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java [hunk 2]
```diff
@@ -384,26 +385,37 @@ else if (posY > maxY - 100)
...
-			// displays the number of points
-			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 40));
-			g.setColor(Color.RED);
-			String str1 = "Points: " + eaten + "!";
-			g.drawString(str1, 20, 35);
-			
-			//	displays the health bar
+			// displays the score bar
+			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 20));
+                        g.setColor(Color.GREEN);
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java]: Can you fix the weird indentation here?

File: src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java [hunk 3]
```diff
@@ -384,26 +385,37 @@ else if (posY > maxY - 100)
...
+			
 			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 20));
 			g.setColor(Color.RED);
-			g.drawString("Health", 30, 90);
 			g.drawRect(20, 95, 350, 20);
 			g.fillRect(20, 95, (int)(350 * health/maxHealth), 20);
 
+			// display the points of health inside the health bar
+			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 20));
+                        g.setColor(Color.WHITE);
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java]: And here.

File: src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java [hunk 4]
```diff
@@ -384,26 +385,37 @@ else if (posY > maxY - 100)
...
+			String heal = "My health: " + health + "/50";
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java]: This should be `"Health: "` to match the other label.

File: src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java [hunk 5]
```diff
@@ -384,26 +385,37 @@ else if (posY > maxY - 100)
...
+			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 20));
+                        g.setColor(Color.GREEN);
+                        g.drawRect(20, 70, 350, 20);
+                        g.fillRect(20, 70, (int)(350 * eaten/50), 20);
 			
+			// displays the number of points
+			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 20));
+			g.setColor(Color.WHITE);
+			String str1 = "Points: " + eaten + "/50";
+			g.drawString(str1, 135, 85);
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java]: This is a little too high up inside the points bar.  Can you make it the same alignment as the text in the health bar?

Comment (yuhaozhang666) [reply src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java]: Already fixed.

File: src/edu/ucsb/cs56/projects/games/fish_animation/Fish.java (hunk 1 above)
Comment (yuhaozhang666) [reply src/edu/ucsb/cs56/projects/games/fish_animation/Fish.java]: Deleted!

File: src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java [hunk 6]
```diff
@@ -384,26 +385,37 @@ else if (posY > maxY - 100)
...
-			// displays the number of points
-			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 40));
-			g.setColor(Color.RED);
-			String str1 = "Points: " + eaten + "!";
-			g.drawString(str1, 20, 35);
-			
-			//	displays the health bar
+			// displays the score bar
+			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 20));
+						g.setColor(Color.GREEN);
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java]: Weird indentation here.

File: src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java [hunk 7]
```diff
@@ -384,26 +385,37 @@ else if (posY > maxY - 100)
...
+			
 			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 20));
 			g.setColor(Color.RED);
-			g.drawString("Health", 30, 90);
 			g.drawRect(20, 95, 350, 20);
 			g.fillRect(20, 95, (int)(350 * health/maxHealth), 20);
 
+			// display the points of health inside the health bar
+			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 20));
+						g.setColor(Color.WHITE);
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java]: And here.

File: src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java [hunk 8]
```diff
@@ -384,26 +385,37 @@ else if (posY > maxY - 100)
...
+			String heal = "My health: " + health + "/50";
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java]: Oops.  One more.  I think I'd said this earlier, but it got lost in an edit.  This should be `"Health: "` to match `"Points: "`.

File: src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java (hunk 6 above)
Comment (yuhaozhang666) [reply src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java]: Already done the three parts!

**Comment by xavierholt** [comment]
Looks good!  I can't merge just yet, though, 'cause Github says there are merge conflicts.  You should resolve those, and then you're good to go.

To do that, first make sure you have an "upstream" remote for the official repo (you can check this with `git remote -v`).  If you don't have one, you can add it with:

    git remote add upstream git@github.com:UCSB-CS56-Projects/cs56-games-fish-animation.git

Once you have it, pull in the changes from the official master branch.  If there are conflicts, Git should tell you how to fix them:

    git pull upstream master

And once you've fixed the conflicts, push to your branch:

    git push

Let me know if you have any trouble!

**Pull request merged**